dev/integration_tests/flavors/ios/Runner/Info-Paid.plist

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is a duplicate of `Info-Free.plist`, but a user could set this up to have different launch screens or whatever.  This just proves they can be different between targets.